src/app/modules/angular-slickgrid/editors/autoCompleteEditor.ts

```diff
@@ -63,13 +65,17 @@ export class AutoCompleteEditor implements Editor {
     return this.columnEditor.validator || this.columnDef.validator;
   }
 
+  get forceUserInputChecker() {
+    return this.columnEditor && this.columnEditor.editorOptions && this.columnEditor.editorOptions.forceUserInput ? this.columnEditor.editorOptions.forceUserInput : false;
+  }
+
   init(): void {
     const columnId = this.columnDef && this.columnDef.id;
     const placeholder = this.columnEditor && this.columnEditor.placeholder || '';
     const title = this.columnEditor && this.columnEditor.title || '';
     this.labelName = this.customStructure && this.customStructure.label || 'label';
     this.valueName = this.customStructure && this.customStructure.value || 'value';
-
+    this.forceUserInput = this.forceUserInputChecker;
```

Comment:
do you really need this line after using the Getter?

Reply:
not after the changes you requested, removed it.